apps/expo-go/ios/Podfile.lock

```diff
@@ -2252,7 +2252,7 @@ CHECKOUT OPTIONS:
 SPEC CHECKSUMS:
   boost: d3f49c53809116a5d38da093a8aa78bf551aed09
   CocoaLumberjack: 2f44e60eb91c176d471fdba43b9e3eae6a721947
-  DoubleConversion: fea03f2699887d960129cc54bba7e52542b6f953
+  DoubleConversion: 76ab83afb40bddeeee456813d9c04f67f78771b5
```

Comment:
Can you please confirm if this is correct? Same for hermes-engine. You can do it by removing Pods folder and reinstall them.
These hashes should change only when we upgrade RN.